### DISCUSSION TITLE
Page not loading.

### DISCUSSION THREAD

**Discussion by tgratton** [Security-Onion-Solutions/securityonion#12761] [Unsupported Versions]
Hello,
When I try to load the security onion SOC the page never loads. I tried assigning it an IP address based on my VM interface and based on my Host address with no luck. I have also tried turning off Windows Defender and the Firewall, and the page still will not load.

**Comment by cm-ops** [comment]
Are you able to ping the SO server from your host machine? If so, do you see your IP in the analyst list of `sudo salt-call pillar.get firewall`?
